stdlib/array.ml

```diff
@@ -30,6 +30,12 @@ external unsafe_blit :
 external create_float: int -> float array = "caml_make_float_vect"
 let make_float = create_float
 
+module Floatarray = struct
```

Comment:
Of these I like `Float.Array` best, since it's easiest to extend to additional types later (e.g. a specialized `Int64.Array` or `Bool.Array`).

Of course, we'd need a `Float` module first.